### PR TITLE
Finished Categories/Dashboard

### PR DESCRIPTION
**Pull request by samirthakral**
### Assignee Tasks

- [ ] created dashboard for admin on navbar
- [ ] can view categories with options for adding, editing and deleting categories
- [ ] added pop-up form for editing using Modal

connects to #21 
connects to #22 
connects to #103

### CODE REVIEW

File: client/components/admin/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        {/* {display === 'Products' && <AdminProducts />} */}
+        {display === 'Categories' && <AdminCategories  />}
+        {/* {display === 'Orders' && <AdminOrders  />} */}
+        {/* {display === 'Users' && <AdminUsers  />} */}
+      </div>
+    )
+  }
+}
+
+const mapStateToProps = (state) => {
```
Comment (maubertw) [inline client/components/admin/dashboard.js:38]: are these empty so that props can be added later?  If not, you should be able to leave out connect.

**Review by maubertw — CHANGES_REQUESTED** [review]
A question about empty map state and map props in dashboard component

File: client/components/admin/dashboard.js (hunk 1 above)
Comment (samirthakral) [reply client/components/admin/dashboard.js:38]: I put them there to use props later but I am not sure if I need them. You guys will also work on the dashboard so I'll leave it for now.

Comment (maubertw) [reply client/components/admin/dashboard.js:38]: 👍

**Review by maubertw — APPROVED** [review]
👍

**Pull request merged**